outputs.tf

```diff
@@ -26,6 +26,10 @@ output "aks_cluster_password" {
   sensitive = true
 }
 
+output "aks_pod_cidr" {
+  value = var.aks_pod_cidr
+}
+
```

Comment:
In the GKE code base we have this - [gke_pod_subnet_cidr](https://github.com/sassoftware/viya4-iac-gcp/blob/main/outputs.tf#L106C9-L106C9) and now we'll have aks_pod_cidr. Would have been better to name the variable `cluster_pod_cidr` to generalize this for each provider. But this will do for now. We'll need another JIRA story around changing and normalizing this.